cookbooks/plume_2D_chunk/doc/plume.tex

```diff
@@ -0,0 +1,98 @@
+
+\textit{This section was contributed by Cedric Thieulot and Paul Bremner.}
+
+This cookbook is inspired by Kellogg \& King (1997) \cite{keki97} but is not an attempt at reproducing the results of their publication. Their study was entirely carried out in dimensionless form. Here, however, we choose to build a similar experiment based on Earth dimensions and material properties.
+Furthermore, in this cookbook we run strictly 2D, whereas the original paper makes use of axisymmetry to restrict the problem to two velocity degrees of freedom \cite{kiha92}.
+
+The two-dimensional domain is a section of an annulus, i.e. a 2D chunk with $\theta \in [3\pi/8:\pi/2]$. Free-slip boundary conditions are imposed on all boundaries. The inner and outer radii are $R\textsubscript{inner}=3480~\si{\km}$ and $R\textsubscript{outer}=6371~\si{\km}$, respectively.
```

Comment:
I find the part about the opening angle a bit confusing. It wasn't clear to me if that was supposed to be a range in the sense of min theta to max theta, or if these were values for max theta for three different models; and then it wasn't clear to me if that was supposed to refer to the original study, which I think uses a colatitude theta 0...pi/4. Can you clarify this?

Reply:
I changed theta to phi, to match the input file. Change it to $ 3\pi/8 \leq \phi \leq \pi/2$